platform-sdk/swirlds-platform-core/src/main/java/com/swirlds/platform/network/connectivity/TlsFactory.java

```diff
@@ -123,4 +107,40 @@ public TlsFactory(
         clientSocket.startHandshake();
         return clientSocket;
     }
+
+    @Override
+    public void handlePeerListUpdate(final @NonNull List<PeerInfo> peers) {
```

Comment:
In some places, this method uses the parameter name `PeerInfoList`, but here it uses the parameter name `peers`. My personal preference would be to use `peers` everywhere. Whichever you choose, probably best to use the same in all places.